packages/moneris-integration/tsconfig.spec.json

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../dist/out-tsc",
+    "module": "commonjs",
+    "types": ["jest", "node"]
+  },
+  "include": [
+    "**/*.test.ts",
+    "**/*.spec.ts",
+    "**/*.test.tsx",
+    "**/*.spec.tsx",
+    "**/*.test.js",
+    "**/*.spec.js",
+    "**/*.test.jsx",
+    "**/*.spec.jsx",
```

Comment:
js and jsx related rows can be removed since we don't use them in this project

Reply:
The coverage for moneris-integration package is meet. Attached screenshot in PR description shows test run executed with `npx nx run moneris-integration:test --coverage`